src/h3/api/_api_template.py

```diff
@@ -10,6 +10,12 @@
 from .. import _cy
 
 
+# H3 address vs H3 cell vs H3 Idnex?
+# do we want a better type system here?
+# how to describe `Set[H3 Index]`, have type distinguish between
+# cells and edges?
```

Comment:
typo: idnex -> index

I'd suggest moving this to an issue here or on uber/h3.

Reply:
Yeah, I will before landing. These are still draft notes :)